### PR TITLE
Iteration 1 - Traffic Control

### PR DESCRIPTION
**Pull request by jeffrade**

### CODE REVIEW

File: traffic_control/api_routes.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+PUT      /api/:version_id/vehicle/:id
+# When I maintenance an airplane, I need to update information on the vehicle
+# Pass JSON with updated info, e.g. { "vehicle" : { "status" : "using", "description" : "foobar" }, ... }
+
+GET      /api/:version_id/flights/search
+# Find flights matching some search criteria
+# Pass JSON with search criteria, e.g. { "search" : { "make" : "boeing", ... }, ...}
+
+GET      /api/:version_id/airport/:code
```
Comment (ssapra) [inline traffic_control/api_routes.txt]: I would try and stay consistent in keeping the endpoint plural. I'd use airports over airport and vehicles over vehicle on line 44.

File: traffic_control/api_routes.txt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+GET      /api/:version_id/flights/:id/
+# See when the airplane will have to refuel during those trips
+
+PUT      /api/:version_id/vehicle/:id
+# When I maintenance an airplane, I need to update information on the vehicle
+# Pass JSON with updated info, e.g. { "vehicle" : { "status" : "using", "description" : "foobar" }, ... }
+
+GET      /api/:version_id/flights/search
+# Find flights matching some search criteria
+# Pass JSON with search criteria, e.g. { "search" : { "make" : "boeing", ... }, ...}
```
Comment (ssapra) [inline traffic_control/api_routes.txt]: Another way I'd approach search is just to pass in all the parameters in the URL especially if it's a GET request

**Comment by ssapra** [comment]
Using the version id in the url is great! I also really like the examples of the data you can pass in; Makes for good documentation.

File: traffic_control/api_routes.txt (hunk 1 above)
Comment (jeffrade) [inline traffic_control/api_routes.txt]: I was actually trying to stay true to that and missed that one :(

File: traffic_control/api_routes.txt (hunk 2 above)
Comment (jeffrade) [inline traffic_control/api_routes.txt]: I meant for that to be a POST.  Will fix.

**Comment by jeffrade** [comment]
@ssapra Thanks Shaan for reviewing.  Will make necessary fixes.

**Comment by paulghaddad** [comment]
@jeffrade Really nice work on this exercise! I really liked the documentation, too. No need to resubmit, but feel free to make changes on your end. :)

**Pull request closed without merging**